packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "16.4.0",
+  "version": "17.0.0",
```

Comment:
This is a breaking change so I'm preemptively bumping core a major version. @Andrew565 We can either wait to merge this PR until our next scheduled release (Wed, July 5th), or, I can revert this and we will just have to remember to make it a major version release on that date instead.

Reply:
Preemptively bumping core here makes sense to me!